packages/grid/_modules_/grid/hooks/features/virtualization/useGridVirtualRows.ts

```diff
@@ -379,30 +379,31 @@ export const useGridVirtualRows = (apiRef: GridApiRef): void => {
     };
   }, []);
 
-  const preventViewportScroll = React.useCallback(
-    (event: any) => {
-      logger.debug('Using keyboard to navigate cells, converting scroll events ');
-
-      event.target.scrollLeft = 0;
-      event.target.scrollTop = 0;
-      event.preventDefault();
-      event.stopPropagation();
-      return false;
-    },
-    [logger],
-  );
+  const preventScroll = React.useCallback((event: any) => {
+    event.target.scrollLeft = 0;
+    event.target.scrollTop = 0;
+    event.preventDefault();
+    event.stopPropagation();
+    return false;
+  }, []);
 
   useNativeEventListener(apiRef, windowRef, GRID_SCROLL, handleScroll, { passive: true });
   useNativeEventListener(
     apiRef,
     () => apiRef.current?.renderingZoneRef?.current?.parentElement,
     GRID_SCROLL,
-    preventViewportScroll,
+    preventScroll,
+  );
+  useNativeEventListener(
+    apiRef,
+    () => apiRef.current?.columnHeadersContainerElementRef?.current,
+    GRID_SCROLL,
+    preventScroll,
   );
   useNativeEventListener(
     apiRef,
     () => apiRef.current?.columnHeadersContainerElementRef?.current?.parentElement,
     GRID_SCROLL,
```

Comment:
This element is not scrollable, also looking at the origin of the regression, I don't see why we would need this.

```suggestion
```

Reply:
I didn't find any place where we change the `scrollLeft` of this element. I suspect it's caused because of the browser's behavior.

Reply:
> This element is not scrollable

https://www.stefanjudis.com/today-i-learned/elements-with-overflow-hidden-are-scrollable/

We could use `overflow: clip`:

![image](https://user-images.githubusercontent.com/42154031/126015927-762050e0-b3c3-4af8-98d3-786140da6f2a.png)

Reply:
To be clear, this line refers to this element: 

<img width="642" alt="Capture d’écran 2021-07-17 à 01 07 35" src="https://user-images.githubusercontent.com/3165635/126016516-a64fb7a3-e498-40c8-89b4-ffbc1518c40d.png">

Reply:
Ohh yes. I completely overlooked this.